targets/targets.json

```diff
@@ -2545,6 +2545,16 @@
         "device_has": ["ANALOGIN", "ERROR_PATTERN", "I2C", "I2C_ASYNCH", "INTERRUPTIN", "LOWPOWERTIMER", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "RTC", "SERIAL", "SERIAL_ASYNCH", "SLEEP", "SPI", "SPI_ASYNCH", "SPISLAVE"],
         "release_versions": ["2", "5"],
         "overrides": {"lf_clock_src": "NRF_LF_SRC_RC"},
+        "config": {
+            "lf_clock_rc_calib_timer_interval": {
+                "value": 16,
+                "macro_name": "MBED_CONF_NORDIC_NRF_LF_CLOCK_CALIB_TIMER_INTERVAL"
```

Comment:
@0xc0170 I follow the instruction stated in \targets\TARGET_NORDIC\TARGET_NRF5\LF_Clock_config.md
I have tried to use “marcos_add” instead of “marco_name”, but the compiled BLE_HeartRate won’t work on DELTA_DFBM_NQ620
Any suggestion?